src/main/java/teammates/ui/webapi/output/StudentsData.java

```diff
@@ -13,7 +13,12 @@
     private final List<StudentData> students;
 
     public StudentsData(List<StudentAttributes> students) {
-        this.students = students.stream().map(StudentData::new).collect(Collectors.toList());
+        this.students = new ArrayList<>();
+        for (StudentAttributes s : students) {
+            StudentData data = new StudentData(s);
+            data.setGoogleId(s.googleId);
+            this.students.add(data);
+        }
```

Comment:
I think it's a better idea to offload this logic into the `StudentData` constructor instead?

Reply:
Main reason why I did this was because other methods use this object as well, but for them, `googleId` may be a field that we want to hide. 
So instead of adding it in the constructor and subsequently hiding it for all current usages, opted for this  less elegant approach because search should be the only instance of `StudentsData` where `googleId` is needed. lmk what you think

Reply:
Ah okay I see your point, let's go with this then! I think the build is failing because it expects `googleId` to be non-null, adding `@Nullable` to the attribute should fix it (refer to `AuthInfo` for example)